apps/web/src/pages/UpcomingLessons.tsx

```diff
@@ -10,24 +10,27 @@ import {
 import { getRateLessonQuery } from "@/lib/query";
 import { useFormatMessage } from "@litespace/ui/hooks/intl";
 import { RatingDialog } from "@litespace/ui/RatingDialog";
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { useNavigate, useSearchParams } from "react-router-dom";
 import { useToast } from "@litespace/ui/Toast";
 import { first } from "lodash";
 import { IUser } from "@litespace/types";
 import { getErrorMessageId } from "@litespace/ui/errorMessage";
 import { capture } from "@/lib/sentry";
+import dayjs from "@/lib/dayjs";
 
 type RateDialogInfo = {
   tutorId: number | null;
   lessonId: number | null;
   tutorName: string | null;
+  start: string | null;
   canRate: boolean;
 };
```

Comment:
The name `RateDialogInfo` is not accurate, we can come up with a better name. 

```suggestion
type RateLessonParams = {
  tutorId: number | null;
  lessonId: number | null;
  tutorName: string | null;
  start: string | null;
  canRate: boolean;
};
```